README.md

```diff
@@ -58,3 +58,68 @@ Starter opp en express-backend som trengs for at bruke applikasjonen.
 ### `npm test`
 
 Kjører alle tester i "watch mode". Ved å trykke på `a`-tasten kjører alle tester. Når testene blir oppdatert vil testene kjøres automatisk.
+
+
+# Oppgaver
+
+
+💡 La applikasjonen kjøre mens jobber på oppgavene, [som beskrevet i denne seksjonen](#starte-applikasjonen). Vær oppmerksom på output i konsolen. Der vil du som regel få informasjon om det som eventuelt ikke fungerer. 
+
+💡 Har du spørsmål? Stuck i oppgaven? Ta kontakt på Slack
+
+
+## Oppgave 1: Testing React komponenter
+
+## Oppgave 2: Mock en modul med `jest.mock`
+
+## Oppgave 3: Mock nettverk kall med `fetch-mock`
```

Comment:
Tänkte du ha några fler subtasks än 3a) eller är denna delen mest för att visa hur man kan mocka nettverkskall i en applikasjon?

 Jag tänker att det kanske är viktigast att lägga fokus på `Oppgave 2: Mock en modul med jest.mock`  så man lär sig hur man kan hantera nettverkskall när man skriver tester?

Reply:
Ja, jeg skal ha flere enn 3a) 
Og, ja også, på oppgave 2) skal vi teste `getCompletion()` som gjør to kall mot `stats/created` og `stats/deleted`
Der skal de først lære å skrive koden som mocker kallene og etterpå skrive funksjonen ... kanskje litt TDD aktig